DQM/TrackingMonitorSource/python/TrackingSourceConfig_Tier0_cff.py

```diff
@@ -178,7 +178,7 @@
     label = 'TrackerCollisionSelectedTrackMonCommon' + str(tracks)
     TrackingDQMSourceTier0 += locals()[label]
 # seeding monitoring
-for era in _cfg.allEras():
+for era in _cfg.allEras() + ["trackingPhase2PU140"]: # FIXME:: allEras() extension should be removed when phase2 tracking is migrated to eras
     postfix = _cfg.postfix(era)
```

Comment:
if it's all so temporary, why edit T0 DQM file setup (I'm guessing it's used only in prompt reco DQM clients)